language-pack/src/main/res/values-zh/menus.xml

```diff
@@ -3,16 +3,16 @@
   <!-- Song Select -->
   <string name="menu_score" formatted="false">成绩：%s (%dx)</string>
   <string name="menu_performance" formatted="false">表现：%spp (%dx)</string>
-  <string name="menu_creator">作者：%s</string>
+  <string name="menu_creator">谱师：%s</string>
   <string name="menu_mod_back">返回</string>
   <string name="menu_mod_multiplier" formatted="false">成绩 %.2fx</string>
   <string name="menu_mod_reset">重置所有 Mod</string>
   <string name="menu_properties_title">歌曲属性</string>
-  <string name="menu_properties_offset">歌曲偏移：</string>
+  <string name="menu_properties_offset">歌曲延时</string>
```

Comment:
Is the removal of the colon intended here? This is used in the modal that is displayed when you hold a beatmap card.